src/main/java/org/spongepowered/api/data/manipulator/mutable/entity/AreaEffectCloudData.java

```diff
@@ -70,34 +75,45 @@
      * with {@link PotionEffect}s.
      *
      * @return The bounded wait time value
+     * @see Keys#AREA_EFFECT_CLOUD_WAIT_TIME
      */
     MutableBoundedValue<Integer> waitTime();
 
     /**
      * Gets the bounded {@code radiusOnUse} value of this cloud.
-     * The radius is reduced per {@link Entity} affected.
+     * The radius is modified per {@link Entity} affected.
      * Once the radius is {@code 0}, the cloud dissipates.
+     * <p>A positive radiusOnUse value will make the cloud grow, a negative
```

Comment:
If you're going to have a paragraph, make a new line between this paragraph and the preceding paragraph.